linetools/isgm/utils.py

```diff
@@ -416,6 +434,26 @@ def table_from_complist(complist):
     assert len(key_order) == len(tab.keys())
     tab = tab[key_order]
 
+    # May need to add an HI component here as done in the method below
+    if NHI_obj is not None:
+        # Existing row in Table?
+        mt = np.where((tab['Z'] == 1) & (tab['ion']==1))[0]
+        if len(mt) == 1:
```

Comment:
Should be matched in redshift as well, there could be several HI components at different redshifts in the complist.

Reply:
I suppose, but not implement for now.